GiniVision/Classes/Core/GiniConfiguration.swift

```diff
@@ -648,10 +648,20 @@ import UIKit
      */
     @objc public var multipagePageIndicatorColor = Colors.Gini.blue
     
+    /**
+     Sets the background color of the page selected indicator
+     */
+    @objc public var multipagePageSelectedIndicatorColor = Colors.Gini.blue
+    
     /**
      Sets the tint color of the page background
      */
     @objc public var multipagePageBackgroundColor = UIColor.white
+    
+    /**
+     Sets the tint color of the draggable icon 
```

Comment:
I'm not sure this description is enough for people to understand what it actually does, but I assume (and hope) the integration guide will help.